src/sentry/integrations/slack/message_builder/notifications/daily_summary.py

```diff
@@ -41,7 +41,7 @@ def __init__(
     def linkify_error_title(self, group):
         link = get_title_link(group, None, False, False, self.notification, ExternalProviders.SLACK)
         title = build_attachment_title(group)
-        attachment_text = self.get_attachment_text(group)
+        attachment_text = self.get_attachment_text(group).replace("\n", " ")
```

Comment:
Does the newline come from the error text itself?

Reply:
Yeah, this is the one I was basing it off of: https://sentry.sentry.io/issues/2733785806/